src/v1.js

```diff
@@ -204,6 +205,11 @@ builder.declare({
     }
   }
 
+  //handle an invalid schema
+  if (!ajv.validateSchema(hookDef.triggerSchema)) {
+    return res.reportError('InputError', 'Invalid Schema', {});
```

Comment:
Should we maybe specifically reject schemas that aren't of the correct version? i.e. draft-6 or something?

Reply:
I think the idea here is to catch most schema errors during the API call. If it turns out that the schema has some other issue, that will appear when triggered. So, I don't think we need to try to do anything fancy (and bug-prone) here.